### ISSUE TITLE
cfn validate is failing and need to fix ASAP

### ISSUE THREAD

**Issue by kaviram1980** [aws-cloudformation/cloudformation-cli#1077]
Hi there,
   cfn validate fails in my pipeline with the following message asking me to create an issue here with rpdk.log file.

> testing cfn issue
> === Unhandled exception ===
> Please report this issue to the team.
> Issue tracker: github.com/aws-cloudformation/cloudformation-cli/issues
> Please include the log file 'rpdk.log'
> === Unhandled exception ===
> Please report this issue to the team.
> Issue tracker: github.com/aws-cloudformation/cloudformation-cli/issues
> Please include the log file 'rpdk.log'

[rpdk.log](https://github.com/user-attachments/files/15917640/rpdk.log)

**Comment by kddejong** [comment]
`Project file C:\Users\kramkumar\IdeaProjects\cloud-infra\.rpdk-config not found. Have you run 'init' or in a wrong directory?`

Do you have the `.rpdk-config` file in the root of the folder you are running cfn validate from?

**Comment by kaviram1980** [comment]
Yes, I am zipping all the necessary files and folders folder contents . . .
fragments
rpdk.log
schema.json

**Comment by kddejong** [comment]
okay.  Looks like you are doing a module.  There was an upgrade to cfn-lint in which the API changed that broke cloudformation-cli.  Try to upgrade cfn-lint to v1.3.2 and see if this issue is resolved.

**Comment by kaviram1980** [comment]
Yeah. It will automatically pull the latest version of cfn-lint 1.3.0 after running the below command
pip3 install cloudformation-cli-python-plugin
Successfully installed Click-8.1.7 Jinja2-3.1.4 Werkzeug-3.0.3 annotated-types-0.7.0 attrs-23.2.0 aws-sam-translator-1.89.0 boto3-1.34.130 certifi-2024.6.2 cfn-flip-1.3.0 cfn-lint-1.3.0
How do I specify the cfn-lint version v1.3.2 when I installing below command
pip3 install cloudformation-cli-python-plugin

**Comment by kddejong** [comment]
Correct try `pip install --upgrade cfn-lint` and then run cfn validate again

**Comment by kddejong** [comment]
On my module projects it is working again so I think you'll be resolved.
